rum/app.py

```diff
@@ -10,41 +11,40 @@ def __init__(self, username, email, access_level='read_only', send_mail=True):
         self.access_level = access_level
         self.sendmail = send_mail
         self.redshift = Redshift()
+        self.check_read_only_exists = open('sqls/check_read_only_exists.sql').read()
+        self.create_user = open('sqls/create_user.sql').read()
+        self.remove_user = open('sqls/remove_user.sql').read()
+        self.add_user_read_only_group = open('sqls/add_user_read_only_group.sql').read()
```

Comment:
You could create a helper function to read these files. Also use the context manager form of the `open` function to ensure that opened file descriptors area closed. Suggestion:

```python
def read_sql_file(filename: str) -> str:
    with open(filename) as fd:
        return fd.read()


class RedshiftUserManagement:
    def __init__(self, **kwargs):
        ...
        self.check_read_only_exists = read_sql_file('sqls/check_read_only_exists.sql')
        ...
```

Reply:
Great! Thanks for noticing that. Created the function under utils folder.